internal/terraform/node_resource_abstract_instance.go

```diff
@@ -2016,9 +2016,25 @@ func (n *NodeAbstractResourceInstance) apply(
 	}
 
 	if !configVal.IsWhollyKnown() {
-		diags = diags.Append(fmt.Errorf(
-			"configuration for %s still contains unknown values during apply (this is a bug in Terraform; please report it!)",
-			n.Addr,
+		// We don't have a pretty format function for a path, but since this is
+		// such a rare error, we can just drop the raw GoString values in here
+		// to make sure we have something to debug with.
+		var unknownPaths []string
+		cty.Transform(configVal, func(p cty.Path, v cty.Value) (cty.Value, error) {
+			if !v.IsKnown() {
+				unknownPaths = append(unknownPaths, fmt.Sprintf("%#v", p))
+			}
+			return v, nil
+		})
+
+		diags = diags.Append(tfdiags.Sourceless(
+			tfdiags.Error,
+			"Configuration contains unknown value",
+			fmt.Sprintf("configuration for %s still contains unknown values during apply (this is a bug in Terraform; please report it!)\n"+
```

Comment:
Is this always a problem in Core, or could it be a provider bug?

Reply:
Yes, this is a core bug in that we should have caught an unknown value before we even get here. Once we've gotten this far it's kind of too late, but hopefully at least showing which part of the config was unknown can help track down the source of the unknown value.